cloud/src/recycler/checker.cpp

```diff
@@ -1758,4 +1766,242 @@ int InstanceChecker::do_mow_job_key_check() {
     return 0;
 }
 
+int InstanceChecker::do_tablet_stats_key_check() {
```

Comment:
print log when the check starts and how many keys scanned, time eplapsed, and the check result (leak loss time takend for each check)